src/open_weather_map/client.rs

```diff
@@ -55,3 +86,15 @@ impl WeatherApiClient {
         Err(ClientError::StatusCodeError((status_code, string_response)))
     }
 }
+
+mod tests {
+
+    #[tokio::test]
+    async fn fetch_weekly() {
+        let weather_client = super::WeatherApiClient::weather_client().await;
```

Comment:
not sure if it's a good idea to make real http requests in tests

Reply:
I will delete this test

Reply:
[Make private decodes and deleting no longer more useful test](https://github.com/pxp9/weather_bot_rust/pull/29/commits/15b1bee39f8105061f5757fec30dbfdcf6bb2433)